subfun/BIDS_RFX.m

```diff
@@ -60,14 +60,14 @@ function BIDS_RFX(action, mmFunctionalSmoothing, mmConSmoothing, opt)
     case 1 % Smooth all con images
 
         matlabbatch = {};
-
+        counter = 0;
```

Comment:
Added a counter for the matlabbatch. 
This will create a new line for each subject --> CON smoothing will occur for all subjects